README.md

```diff
@@ -236,6 +236,8 @@ Additionally, several packages that include the VSCode version of some services
   - Survey/feedback support
 - **Update**
   - Update detection, release notes...
+- **Secret Storage**
+  - Storage of secrets for extensions, will store by default in-memory, unless passed a custom implementation in the environment service.
```

Comment:
`unless passed a custom implementation in the environment service` is probably a little bit confusing, are you talking about the WorkbenchContructionOptions given to the `initialize` function?

Reply:
Yep, I'll make this more clear.